pkg/resourcemanager/webhook/vpainplaceorrecreateupdatemode/handler_test.go

```diff
@@ -0,0 +1,71 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package vpainplaceorrecreateupdatemode
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	vpaautoscalingv1 "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1"
+	"k8s.io/utils/ptr"
+	logzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/gardener/gardener/pkg/logger"
+)
+
+var _ = Describe("Handler", func() {
+	var (
+		ctx     = context.TODO()
```

Comment:
This assignment does does not bring much value as in the BeforeEach block the ctx is always re-assigned.

```suggestion
		ctx     context.Context
```

Reply:
Thanks for the suggestion, removed the _unit_ tests in favour of the _integration_ tests. :))